docs/usage.md

```diff
@@ -30,6 +30,35 @@ does not load classes from file, then you must use `SourceLocator` creation.
 *Fun fact... using `ReflectionClass::createFromName` actually uses a
 SourceLocator under the hood - it uses the `AutoloadSourceLocator`.*
 
+### Initialisers
+
+There are several static initialisers you may use based on the same concept.
+They are as follows:
+
+```php
+<?php
+
+use BetterReflection\Reflector\ReflectionClass;
+use BetterReflection\Reflector\ReflectionMethod;
+use BetterReflection\Reflector\ReflectionParameter;
+use BetterReflection\Reflector\ReflectionProperty;
+
+ReflectionClass::createFromName(\stdClass::class);
+ReflectionClass::createFromName(new \stdClass);
```

Comment:
do we support `createFromName` via objects? Should we instead have `createFromInstance`?